### PR TITLE
[TASK] Migrate to TYPO3 10.4

### PR DESCRIPTION
**Pull request by chriwo**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
     {
       "name": "Hannes Lau",
       "email": "office@hanneslau.de"
     }
   ],
-  "license": "GPL-2.0+",
-  "type": "typo3-cms-extension",
   "require": {
-	"pti/pti": "*",
+    "php" : "^7.4",
```
Comment (SomeBdyElse) [inline composer.json]: fix indent

File: composer.json [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+	"pti/pti": "^3.0",
 	"twig/twig": "^2.0",
-  	"typo3/cms-core": "^9.5"
-  },
-  "replace": {
-    "pti/pti-twig": "self.version",
-    "typo3-ter/pti-twig": "self.version"
+  	"typo3/cms-core": "^10.4"
```
Comment (SomeBdyElse) [inline composer.json]: fix indent

**Review by SomeBdyElse — APPROVED** [review]
Seems fine, only two indents look strange in github's diff view

File: ext_emconf.php [hunk 3]
```diff
@@ -2,4 +2,11 @@
 $EM_CONF['pti_twig'] = [
     'title' => 'PTI Twig',
     'description' => 'Prototype Integration - Twig',
+    'version' => '3.0',
```
Comment (SomeBdyElse) [inline ext_emconf.php]: what happens if we leave the version number unset in ext_emconf? It would be nicer to do this via git tags only.

**Comment by SomeBdyElse** [comment]
I updated the PR title. Please use the new title as the commit message.

File: ext_emconf.php (hunk 3 above)
Comment (chriwo) [reply ext_emconf.php]: I have removed the version

**Comment by SomeBdyElse** [comment]
Thanks for removing the version. Please check the indents commented above. If you consider them fine, feel free to merge.

**Review by SomeBdyElse — APPROVED** [review]

**Pull request merged**